### DISCUSSION TITLE
Any roadmap on new supported servers?

### DISCUSSION THREAD

**Discussion by bhj16** [qdm12/gluetun#352] [Q&A] [answered]
Currently using gluetun to route a lot of my stack through Mullvad, so thanks!

I'm actually moving over to IVPN and I wondered if there was a request system for new supported servers or a roadmap for this? Perhaps just public opinion?

Really appreciate the effort on gluetun.

**Comment by qdm12** [comment]
No roadmap but I'll add an issue template for it.

The thing is I need a bunch of information to support a new vpn provider especially:

1. all their openvpn files (see what options they use, what tls certificates they have)
2. a public http endpoint to get all their server hostnames/IPs somehow (or a zip file of ovpn files) so the server information can be automagically updated over time by the users.

And unfortunately not all VPN providers offer this, Mullvad and Nordvpn being the most opened on this and Cyberghost being the worst for example 😅 (had to DNS resolve all the combinations of country codes to find all their servers lol)

**Comment by qdm12** [comment, marked as answer]
Please use https://github.com/qdm12/gluetun/issues/new/choose (last template) for IVPN. I'm currently adding more VPN providers (fastestvpn, ~torguard~) so it might be interesting for you to create the issue now so I can do it in my current workflow 😉

**Comment by qdm12** [comment]
See #395 I'll try to add in the coming days.
